code/src/index.css

```diff
@@ -1,13 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Serif:ital,wght@1,700&display=swap');
+
+* {
+  font-family: 'PT Serif', serif;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  -moz-osx-font-smoothing: grayscale; 
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  monospace;
+}
+
+.todo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;  
+  background-repeat: no-repeat;
+  background: rgb(255,255,255);
+  background: linear-gradient(0deg, rgba(255,255,255,1) 0%, rgba(201,254,255,1) 50%, rgba(255,255,255,1) 100%);
+}
+
+.todo-form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background: rgb(34,182,213);
+  background: linear-gradient(133deg, rgba(34,182,213,1) 18%, rgba(34,35,213,1) 60%);
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+.counter-container {
+  border-bottom: 1px solid rgb(53, 11, 240);
+  padding: 0 50px;
+}
+
+.counter-container p{  
+  margin: 3px 0;
+}
+
+.todo-form input[type="text"] {
+  margin-top: 20px; 
+  border-radius: 7px;
+  width: 200px;
+  height: 30px;
+}
+
+.checkbox-complete input[type=checkbox] {
+  margin: 0;
+  cursor: pointer;  
+}
+
+.item-container {
+  box-shadow: 0px 0px 5px rgb(211, 211, 211);
+  margin-top: 10px;
+}
+
+.todo-item {
+  display: grid;
+  grid-template-columns: 150px 100px 100px; 
+  grid-column-gap: 5px;   
```

Comment:
Really nice that you use grid! Unfortunately, this one is a little too wide for small mobile screens and overflows out of the app.

Reply:
😢